persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/ClinicalDataMyBatisRepository.java

```diff
@@ -21,32 +20,94 @@
     private OffsetCalculator offsetCalculator;
 
     @Override
-    public List<SampleClinicalData> getAllClinicalDataOfSampleInStudy(List<String> studyIds, List<String> sampleIds,
+    public List<ClinicalData> getAllClinicalDataOfSampleInStudy(String studyId, String sampleId,
```

Comment:
Web API itself didn't change, I just changed the persistence interface of Clinical Data, because it was not consistent with other APIs, I made it more verbose through having more methods.